backend/api/Services/MapService.cs

```diff
@@ -77,7 +77,7 @@ await foreach (var blobPage in resultSegment)
                             boundaries.Add(blobItem.Name, ExtractMapMetadata(blobItem));
                             imageSizes.Add(blobItem.Name, ExtractImageSize(blobItem));
                         }
-                        catch (FormatException)
+                        catch (Exception e) when (e is FormatException || e is KeyNotFoundException)
```

Comment:
Why do we just ignore and continue in the case of these exceptions? Should we not even log a warning?